ajax/libs/jquery.gray/package.json

```diff
@@ -4,16 +4,31 @@
   "version": "1.4.4",
   "description": "Make an image gray in all browsers",
   "license": "MIT",
-  "author": "kununu",
+  "author": "Karl Horky <karl.horky@gmail.com>",
   "keywords": [
     "gray",
     "grey",
     "grayscale",
     "images",
-    "image"
+    "image",
+    "jquery",
+    "jquery-plugin"
   ],
   "repository": {
     "type": "git",
     "url": "git://github.com/karlhorky/gray.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/karlhorky/gray.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "css/*",
+          "js/*"
```

Comment:
I think it will be more flexible to use `css/**/*` and `js/**/*`.
Just in case the author adds directory in them, and we still can fetch the files.